curl/curl.go

```diff
@@ -76,6 +83,19 @@ func (c *Curl) Absorb(inn Trytes) {
 		copy(c.State, in[i:i+lenn])
 		c.Transform()
 	}
+	return nil
+}
+
+// AbsorbTrytes fills the internal State of the sponge with the given trytes.
+// It panics if the given trytes are not valid.
+func (c *Curl) AbsorbTrytes(inn Trytes) {
```

Comment:
Same as SqueezeTrytes: please return an error or provide two methods where one is prefixed with "Must".